src/when.js

```diff
@@ -31,7 +31,7 @@ class WhenMock {
       // * call mocks with equal matchers are removed
       // * `once` mocks are used prioritized
       this.callMocks = this.callMocks
-        .filter((callMock) => once || !utils.equals(callMock.matchers, matchers))
+        .filter((callMock) => once || callMock.once || !utils.equals(callMock.matchers, matchers))
```

Comment:
Without this, if `mockReturnValueOnce` is before `mockReturnValue` it will get removed.

Reply:
To call `mock*ValueOnce` _after_ a `mock*Value` call didn't cross my mind before, but it makes sense. Like it!